src/main/java/com/auction/retailService/constant/ErrorMessageConstant.java

```diff
@@ -0,0 +1,18 @@
+package com.auction.retailService.constant;
+
+public class ErrorMessageConstant {
```

Comment:
if this class is already inside a `constant` package it is kinda redundant to name it `ErrorMessageConstant`, naming it `ErrorMessages` instead, makes more sense